s3/libirods_s3.cpp

```diff
@@ -673,15 +675,15 @@ S3UriStyle s3_get_uri_request_style( irods::plugin_property_map& _prop_map)
 // returns the upper limit of the MPU chunk size parameter, in megabytes
 // used for validating the value of S3_MPU_CHUNK
 // also used for determining the maximum size for CopyObject
-long s3GetMaxUploadSizeMB (irods::plugin_property_map& _prop_map)
+int64_t s3GetMaxUploadSizeMB (irods::plugin_property_map& _prop_map)
 {
     irods::error ret;
     std::string max_size_str;   // max size from context string, in MB
 
     ret = _prop_map.get<std::string>(s3_max_upload_size, max_size_str);
     if (ret.ok()) {
         // should be between 5MB and 5TB
-        long max_megs = atol(max_size_str.c_str());
+        int64_t max_megs = atol(max_size_str.c_str());
```

Comment:
`atol` -> `std::atol` 

Include the `std` namespace for C functions as well.

Reply:
done for all atol